src/values.rs

```diff
@@ -475,15 +476,7 @@ impl Lerpable<Color> for Color {
         // copies the entire struct every time, whereas this should only copy once each.
         // This whas showing up in the hot path when profiling the `basic` example when
         // calling each individually, due to the excessive copies.
-        let rgba = self.as_rgba_f32();
-        let other_rgba = other.as_rgba_f32();
-
-        Color::rgba(
-            rgba[0].lerp(other_rgba[0], clamped_pct),
-            rgba[1].lerp(other_rgba[1], clamped_pct),
-            rgba[2].lerp(other_rgba[2], clamped_pct),
-            rgba[3].lerp(other_rgba[3], clamped_pct),
-        )
+        (*self..other).at(clamped_pct)
```

Comment:
The comment above this line is no longer applicable and can be removed.

Reply:
Removed